terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf

```diff
@@ -174,42 +175,42 @@ module "cucumber_testsuite" {
         memory = 2048
         vcpu = 2
       }
-      additional_packages = [ "venv-salt-minion" ]
-      install_salt_bundle = true
+//      additional_packages = [ "venv-salt-minion" ]
+//      install_salt_bundle = true
     }
     debian-minion = {
       name = "min-ubuntu2204"
-      image = "ubuntu2204o"
+      image = "ubuntu2004o"
       provider_settings = {
         mac = "aa:b2:93:01:00:4b"
       }
-      additional_packages = [ "venv-salt-minion" ]
-      install_salt_bundle = true
+//      additional_packages = [ "venv-salt-minion" ]
+//      install_salt_bundle = true
     }
     build-host = {
-      image = "sles15sp4o"
+      image = "sles15sp3o"
       name = "min-build"
       provider_settings = {
         mac = "aa:b2:93:01:00:4d"
         memory = 2048
         vcpu = 2
       }
-      additional_packages = [ "venv-salt-minion" ]
-      install_salt_bundle = true
+//      additional_packages = [ "venv-salt-minion" ]
+//      install_salt_bundle = true
     }
     pxeboot-minion = {
-      image = "sles15sp4o"
-      additional_packages = [ "venv-salt-minion" ]
-      install_salt_bundle = true
+      image = "sles15sp3o"
+//      additional_packages = [ "venv-salt-minion" ]
+//      install_salt_bundle = true
     }
     kvm-host = {
-      image = "sles15sp4o"
+      image = "sles15sp3o"
       name = "min-kvm"
       provider_settings = {
         mac = "aa:b2:93:01:00:4e"
       }
-      additional_packages = [ "venv-salt-minion" ]
-      install_salt_bundle = true
+//      additional_packages = [ "venv-salt-minion" ]
```

Comment:
Salt bundle is now the default for 4.2.11, are you sur you want to remove this?

Reply:
I adjusted this according to https://github.com/SUSE/susemanager-ci/blob/master/terracumber_config/tf_files/SUSEManager-4.2-NUE.tf where there is no salt bundle.

In any case, in 4.2.11 salt bundle is the default (if available in bootstrap repo) for new bootstrapped client. But normal salt would also work.